astropy/coordinates/distances.py

```diff
@@ -127,10 +127,9 @@ def __new__(
                 "`parallax` were given to Distance constructor"
             )
 
-        if value is None:
-            # If something else but `value` was provided then a new array will
-            # be created anyways and there is no need to copy that.
-            copy = False
+        # If something else but `value` was provided then a new array will
```

Comment:
Here, I like what was there better: just more explicit.

Reply:
I wouldn't have made the change if I didn't think it was an improvement, but it's not worth arguing over.